ui/app/appointments/controllers/manage/appointmentsCreateController.js

```diff
@@ -93,14 +93,49 @@ angular.module('bahmni.appointments')
                 }
             };
 
+            var setAppointmentsEndTimeAmPm = function (date) {
+                var strTime = moment(date).add(30, 'm').toDate();
```

Comment:
@rubailly '30' in here corresponds to the appointment duration? I'm not sure if the value should be hard-coded; could you please check if there is a variable storing the minimum duration that we can use here?